File: android/src/main/java/dragon3/data/StageData.java
/*
 * 作成日: 2004/03/07
 */
package dragon3.data;

import dragon3.stage.StageBack;

/**
 * @author k-saito
 */
@lombok.Data
public class StageData implements Data {

	private String id = "none";
	private String name = "none";
	private int level;
	private StageBack back = StageBack.WHITE;

	@Override
	public String toString(){
		return id + " - " + name;
	}
}

File: core/src/main/kotlin/dragon3/data/StageData.kt
/*
 * 作成日: 2004/03/07
 */
package dragon3.data

import dragon3.stage.StageBack

/**
 * @author k-saito
 */
class StageData : Data {

    override var id = "none"
    override var name = "none"
    var level: Int = 0
    var back = StageBack.WHITE

    override fun toString(): String {
        return id + " - " + name
    }
}
